node/full.go

```diff
@@ -0,0 +1,393 @@
+package node
```

Comment:
We need to rebase the changes somehow, to keep git history of this file (this should be visible as node.go renamed to full.go.

Reply:
So it seems it's a GitHub glitch... because individual commits shows proper rename, and git blame works fine.